Sources/SwiftProtobuf/TextDecoder.swift

```diff
@@ -0,0 +1,211 @@
+// ProtobufRuntime/Sources/Protobuf/ProtobufTextDecoding.swift - Text format decoding
```

Comment:
I just finished updating the leading comments in all the files across the project, and did a separate pass over this branch to catch the files added here.